pkg/logging/logging.go

```diff
@@ -37,6 +37,26 @@ const (
 	Tag        = "tag"
 )
 
+var FluentdLogOpts = []string{
+	fluentAddress,
+	fluentdAsync,
+	fluentdBufferLimit,
+	fluentdRetryWait,
+	fluentdMaxRetries,
+	fluentdSubSecondPrecision,
+	fluentdAsyncReconnectInterval,
+	fluentRequestAck,
+}
+
+var JSONDriverLogOpts = []string{
+	MaxSize,
+	MaxFile,
+}
+
+var JournalDriverLogOpts = []string{
+	Tag,
+}
+
```

Comment:
each `XXLogOpts` should be moved in `XX_logger.go`

Reply:
OK